BatchAI. Updated python quickstart

AlexanderYukhanov opened the pull request:
1. Fixed the code to be compatible with new and old azure-mgmt-batchai
2. Added code to list generated model files
3. Added code to delete created resource group

azurecla commented:
Hi __@vt-cezary__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@vt-cezary : Thanks for your contribution to the Azure documentation! The author, @krisragh, has been notified to review your proposed change.

krisragh commented:
@vt-cezary thanks for your contribution, and good catch on the updated link. I appreciate your update! I notice that the new link you have (changelog.md) doesn't actually have a download link on that page. 

How about updating that link to https://github.com/Azure/azure-mobile-apps-ios-client/blob/master/README.md#ios-client-sdk instead? (I don't have permissions to do this since you opened this PR.)

krisragh commented:
@vt-cezary looks good, thanks for the update. @ggailey777 and @tysonn it's OK to go ahead and merge this.

ggailey777 commented:
@vt-cezary Thanks for helping to improve the quality of Azure docs.

Commit:
BatchAI. Updated python quickstart

1. Fixed the code to be compatible with new and old azure-mgmt-batchai
2. Added code to list generated model files
3. Added code to delete created resource group

llidev commented on articles/batch-ai/quickstart-python.md:
```diff
@@ -134,124 +127,114 @@ Create a Batch AI cluster. In this example, the cluster consists of a single STA
-    ), 
-) 
-batchai_client.clusters.create(resource_group_name, cluster_name, parameters).result() 
+        mount_volumes=models.MountVolumes(
```
should we use job level mounting here?

AlexanderYukhanov replied:
decided it to be out of scope
